developing-with-nats/receiving/structure.md

```diff
@@ -7,7 +7,10 @@ For example, to receive JSON you could do:
 {% tabs %}
 {% tab title="Go" %}
 ```go
-nc, err := nats.Connect("demo.nats.io")
+nc, err := nats.Connect("demo.nats.io",
+        nats.ErrorHandler(func(nc *nats.Conn, s *nats.Subscription, err error) {
+            log.Printf("Async error in %q/%q: %v", s.Subject, s.Queue, err)
```

Comment:
The ErrorHandler can be invoked also for error that do not involve the subscription, for instance a publish violation, etc.. so in other words, `s` may be `nil` here, which would then cause a panic.

Reply:
Oops, I only looked at testing that side when I was using `s` not `s.Subject` when I tried to make it prettier.  Sorry.  Fixed.